CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## PyAdditive 0.18.dev1
```

Comment:
```suggestion
## PyAdditive 0.18.0
```
This doesn't need to change. These will be the release notes when we create the 0.18.0 release.